scripts/deploy-aws.sh

```diff
@@ -11,17 +11,19 @@ if [[ -n "$missing" ]]; then
     exit 1
 fi
 
-CONTROLLER=${CONTROLLER:-aws-kf-controller}
-CLOUD=${CLOUD:-aws-kf-cloud}
-MODEL=${MODEL:-aws-kf-model}
+CONTROLLER=${CONTROLLER:-cdk}
```

Comment:
The `cdk` controller default feels too generic to me. I don't know of other K8s CI jobs that would conflict off the top of my head, but I know I have a personal `cdk` controller that I'd be miffed at losing when I called this script to tear down my kubeflow.

```bash
export CONTROLLER=test-controller
./deploy-aws

## switch to new terminal
./deploy-aws --destroy  # <-- doh!  there goes my 'cdk' controller
```

Suggest tacking on some kind of unique suffix (something simple like `date +%s` would suffice).

Reply:
How about `kfcdk`? I was going for fewer characters to type since I have to type the name quite frequently, but if you feel strongly about the longer name, can use that instead.

Reply:
`kfcdk` is fine, but so is `cdkkf`, and if you do the latter, you're finger is already on the `k` to hit that second one ;)

My main concern is just a selfish one in that i already have `cdk` controller that i might not want to use when trying out kubeflow. Whatever you do to change it up a little is fine by me.